graylog2-server/src/main/java/org/graylog/failure/FailureSubmissionService.java

```diff
@@ -181,4 +199,21 @@ private IndexingFailure fromIndexingError(IndexingError indexingError) {
                 indexingError.index()
         );
     }
+
+    private void updateProcessingFailureMetric(Message message) {
+        Object inputId = message.getField(FIELD_GL2_SOURCE_INPUT);
+        if (inputId != null) {
+            final String indexingFailureMetricName = name("org.graylog2", inputId.toString(), "failures.processing");
```

Comment:
Just a small thing. Perhaps the prefix of the metric should be changed to `org.graylog2.inputs`, like the other input metrics.![image](https://github.com/user-attachments/assets/95d81b73-3dfe-41a3-9796-578f2efba0c1)